### DISCUSSION TITLE
Can Duckdb use multiple characters as separators when reading log and txt files? After testing, it was found that only one symbol can be written

### DISCUSSION THREAD

**Discussion by zhbdesign** [duckdb/duckdb#15459] [General]
Can Duckdb use multiple characters as separators when reading log and txt files? After testing, it was found that only one symbol can be written

**Comment by szarnyasg** [comment]
Hi, do you mean the CSV reader (`read_csv`) component of DuckDB? This current only supports a single delimiter. If you would like to split against multiple separators, I recommend using the [`read_text` function](https://duckdb.org/docs/guides/file_formats/read_file) and splitting along a regular expression using [`regex_split_to_array`](https://duckdb.org/docs/sql/functions/char#regexp_split_to_arraystring-regex).

**Reply by zhbdesign** [reply]
I want to use duck db to read log files, txt files, and the delimiter is @|@

**Reply by pdet** [reply]
The CSV Reader will support up to 4-byte delimiters in the next version.

**Reply by zhbdesign** [reply]
When will this plan be released
